src/definitions/modules/calendar.js

```diff
@@ -116,7 +116,7 @@ $.fn.calendar = function(parameters) {
               module.set.maxDate($module.data(metadata.maxDate));
             }
             module.setting('type', module.get.type());
-            module.setting('on', settings.on || ($input.length ? 'focus' : 'click'));
+            module.setting('on', settings.on || 'click');
```

Comment:
If "click" is "always" expected, I think "click" should be added regardless of settings, something like:


```suggestion
            if (settings.on) module.setting('on', settings.on);
            if (settings.on.indexOf('click') >= 0) module.setting('on', 'click'); // avoid duplicate click
```

Reply:
Sorry, maybe it's too late for me today 😴 , but I don't get the point of your suggested change..
```javascript
if (settings.on) module.setting('on', settings.on);
```
is totally senseless IMHO, because it's equal to `settings.on = settings.on` 😉 

```javascript
if (settings.on.indexOf('click') >= 0) module.setting('on', 'click');
```
What does this do/prevent? the only valid options are 'click', 'hover' and 'focus' anyway.

Reply:
`settings.on` is not a required parameter for calendars. However, the module needs `module.setting('on')` to be set in order to know what event it should listen to.

I think that, with this change, if the user doesn't include `on: (...)` to his calendar definition then the calendar will have no `module.setting('on')` which will end up breaking. (The docs themselves don't use the `on` option, so IMHO doing this would end up breaking the docs even more.

With my commit, the user could have his on option set to `null`, `undefined` or `false` and it would set it to the default: `click`. That's how I envisioned it, `click` should not be expected, just the default.

Reply:
Oh, sorry, please forget my above comment !!

I reazlized that I misunderstood what `settings.on` is.
I just looked the surface of changes from mobile GitHub app... but now I browsed code on IDE and docs.